src/swap/alice.rs

```diff
@@ -95,6 +95,10 @@ where
     }
 }
 
+/// Nectar does not care if the taker refunds, so we do not need to
+/// look for the taker's refund event the blockchain.
```

Comment:
I mean, yes. I was just trying to explain why the `WatchOnly` implementation of Alice refunding Alpha in Nectar is a no-op. In `cnd` this would not be the case.